src/commands/chatters/winter.ts

```diff
@@ -0,0 +1,22 @@
+import { Discord, SimpleCommand, SimpleCommandMessage, SimpleCommandOption, SimpleCommandOptionType } from 'discordx'
+
+@Discord()
+class Winter {
+  @SimpleCommand({ name: 'winter', description: 'Winter', argSplitter: '\n' })
+  async simple(
+    @SimpleCommandOption({ name: 'text', type: SimpleCommandOptionType.String }) text: string | undefined,
+    command: SimpleCommandMessage
+  ) {
+    let content
+    if (text !== undefined) {
```

Comment:
Strings are truthy, this will handle empty strings, null and undefined.
```suggestion
    if (text) {
```